incubator/chartmuseum/values.yaml

```diff
@@ -14,3 +14,20 @@ resources:
   requests:
     cpu: 80m
     memory: 64Mi
+persistence:
+  Enabled: true
```

Comment:
As others would want to customize to their needs, I think this should be defaulted to `false`. Deciding to use `true` here would favor the use of pcvs over other storage mediums like object storage. There will only ever be one storage active at a time. What do you think ?

Reply:
yeah makes sense, done